newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -3,8 +3,8 @@ import optionalRequire from '../Utils/OptionalRequire.js';
 const fs = optionalRequire('fs');
 
 export default class LocalProjectWriter {
-  static _writeProjectJSONFile = (project, filepath, cb) => {
-    if (!fs) return cb('Not supported');
+  static writeProjectJSONFile = (project, filepath, cb) => {
+    if (!fs || !filepath) return cb('Not supported');
```

Comment:
Do you have any reason in mind about why filepath would not be specified or be empty?

Reply:
I think this was a misguided attempt to make it pass flow, but I forgot it in there